README.md

```diff
@@ -16,6 +16,12 @@ Spruce is a lightweight animation library that helps choreograph the animations
 Add the following to your project's build.gradle file
 
 ```
+repositories {
+    maven {
+        url  "http://willowtreeapps.bintray.com/maven2" 
```

Comment:
If jcenter doesn't work that kinda defeats the purpose of uploading it to bintray